core/contracts/oracle/implementation/Voting.sol

```diff
@@ -388,15 +394,15 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface {
             keccak256(abi.encodePacked(price, salt, msg.sender, time, roundId, identifier)) == voteSubmission.commit,
             "Revealed data != commit hash"
         );
-        delete voteSubmission.commit;
 
-        // Lock in round variables including snapshotId and inflation rate. Not that this will only execute a snapshot
-        // if the `snapshotCurrentRound` function was not already called for this round.
-        _freezeRoundVariables(roundId);
```

Comment:
So `revealVote` will no longer freeze round variables and now `snapshotCurrentRound` MUST be called before revealing?

Reply:
yes that is right. this could be changed but requires an optional signature param for revealVote as well as batchReveal and whatever else would need that.